implementation/endpoints/include/netlink_connector.hpp

```diff
@@ -20,6 +20,28 @@
 
 #include "../../endpoints/include/buffer.hpp"
 
+inline bool operator<(const sockaddr_nl &rhs, const sockaddr_nl &lhs)
+{
+    if (rhs.nl_family < lhs.nl_family &&
```

Comment:
It's a style preference but persnally I would write:  return (boolean expression) instead of the redundant "if/else return true/false".  Maintainer decides.

Reply:
I don't mind either way, your way is cleaner so switched over to it.